app/src/main/java/com/example/triptracker/view/map/AddSpot.kt

```diff
@@ -230,6 +229,10 @@ fun FillAddSpot(
           var pos by remember { mutableStateOf(LatLng(0.0, 0.0)) }
           val placeHolderError by remember { mutableStateOf("Please enter a valid location") }
 
+          fun textColor() =
```

Comment:
maybe we could make this function private as it seems like it will only be used in here

Reply:
The private modifier is not applicable to a local function. So it's already private !